src/template_compiler.erl

```diff
@@ -355,16 +355,14 @@ compile_forms(Filename, Forms) ->
                 {module, _Module} ->
                     {ok, Module};
                 Error ->
-                    ?LOG_ERROR("Error loading compiling forms for ~p: ~p",
-                                [Filename, Error]),
+                    ?LOG_ERROR(#{ text => "Error loading compiling forms.", filename => Filename, error => Error}),
                     Error
             end;
         error ->
-            ?LOG_ERROR("Error compiling forms for ~p", [Filename]),
+            ?LOG_ERROR(#{ text => "Error compiling forms.", filename => Filename}),
```

Comment:
In the other logging I always added `result => error`  and then, if known, `reason => ...`